examples/imagefilter/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "imagefilter"
+version = "0.1.0"
+authors = ["SixtyFPS <info@sixtyfps.io>"]
+edition = "2018"
+publish = false
+license = "GPL-3.0-only"
+
+[[bin]]
+path = "main.rs"
+name = "imagefilter"
+
+[dependencies]
+sixtyfps = { path = "../../api/sixtyfps-rs" }
+image = "0.23.14"
+
+[build-dependencies]
+sixtyfps-build = { path = "../../api/sixtyfps-rs/sixtyfps-build" }
+
+# Remove the `#wasm#` to uncomment the wasm build.
+# This is commented out by default because we don't want to build it as a library by default
+# The CI has a script that does sed "s/#wasm# //" to generate the wasm build.
+
+#wasm# [lib]
+#wasm# path = "main.rs"
+#wasm# crate-type = ["cdylib"]
+
+#wasm# [target.'cfg(target_arch = "wasm32")'.dependencies]
+#wasm# wasm-bindgen = { version = "0.2" }
+#wasm# web-sys = { version = "0.3", features=["console"] }
+#wasm# console_error_panic_hook = "0.1.5"
+#wasm# getrandom = { version = "0.2.2", features = ["js"] }
```

Comment:
do we need getrandom or is that just copy/paste?

Reply:
Well spotted, that's copy & pasted. I had removed it locally many times but never committed because I would have accidentally committed without the `#` :).